src/main/java/focusedCrawler/target/TargetStorageConfig.java

```diff
@@ -14,6 +14,10 @@
     
     @JsonProperty("target_storage.target_directory")
     private String targetStorageDirectory = "data_pages";
+
+    @JsonProperty("target_storage.nonHTML_directory")
+    private String nonHTMLStorageDirectory = targetStorageDirectory+"/nonHTML";
```

Comment:
One directory inside another repository directory will create problems for some data formats.